src/core_modules/capture-core/components/Pages/epics/resetProgramAfterSettingOrgUnitIfApplicable.epic.js

```diff
@@ -38,6 +39,8 @@ export const resetProgramAfterSettingOrgUnitIfApplicableEpic = (action$: InputOb
     // $FlowSuppress
     action$
         .ofType(
+            lockedSelectorActionTypes.SET_ORG_UNIT,
+
```

Comment:
Sorry about these spaces. I left them as mark to myself for the next PR that follows so that I can distinguish better `lockedSelector` related stuff.